pages/common/fitstopnm.md

```diff
@@ -0,0 +1,13 @@
+# fitstopnm
+
+> Convert a Flexible Image Transport System (FITS) file into a PNM image.
+> See also: `pamtofits`.
+> More information: <https://netpbm.sourceforge.net/doc/fitstopnm.html>.
+
+- Convert a FITS file into a PNM image:
+
+`fitstopnm {{path/to/file.fits}} > {{path/to/output.pnm}}`
+
+- Convert the image on the specified position of the third axis in the FITS file:
+
+`fitstopnm -image {{n}} {{path/to/file.fits}} > {{path/to/output.pnm}}`
```

Comment:
I think it's possible to use a more descriptive placeholder than "n"

Reply:
IMO it's clear enough for most users, as "n" can also mean an arbitrary real number for some people. I still think that a descriptive placeholder would be better.